src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,17 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <header className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+      <p>
+        {' Posted by '}
+        <UserInfo user={user} />
```

Comment:
let's render userInfo only when the user exists
```suggestion
        {user && <UserInfo user={user} />}
```